readthedocs/doc_builder/config.py

```diff
@@ -68,7 +70,7 @@ def load_yaml_config(version):
         config = BuildConfig(
             env_config=env_config,
             raw_config={},
-            source_file='empty',
+            source_file=path.join(checkout_path, 'empty'),
```

Comment:
In the future, we want to manage this in a less hacky way (I mean the `empty` file)

Reply:
It seems like we'd want to just pass `None` rather than 'empty' but possibly that is a larger refactor.